File: Day 4/PYTHON/perfect_square.py
test_list = [4, 9, 25, 49, 100, 6]
def perfect_square(test_list):
	count = 0
	newBox = test_list
	for number in test_list:
		new_number = number**0.5 * number**0.5
		if new_number == number:
			newBox[count]= True
		else:
			newBox[count]= False
		count += 1
	return newBox

			
print(perfect_square(test_list))

File: Day 4/JAVASCRIPT/PerfectSquare.js
testList = [4, 9, 25, 49]

function perfectSquare(testList){

	let newBox = []
	let number1 =0
	for(let number = 0; number <=testList.length; number++){
		if (number == number**0.5 * number**0.5){
			newBox[number] = true
			return newBox 
		}else{
			newBox[number] = false
			return newBox 	
		}
	}
}
console.log(perfectSquare(testList))





